.travis.yml

```diff
@@ -30,13 +30,12 @@ before_install:
   - ./miniconda.sh -b -p ./miniconda
   - export PATH=`pwd`/miniconda/bin:$PATH
   - conda update --yes conda
-  - conda config --add channels conda-forge
+  - conda config --append channels conda-forge
   - conda create -y -q -n test-env python=$TRAVIS_PYTHON_VERSION
   - source activate test-env
```

Comment:
Can you try to update this to `conda activate test-env` instead?

Reply:
`condo activate` [appears to kill the build](https://travis-ci.org/pysal/spaghetti/jobs/430635191#L437-L618)

Reply:
That is odd. You are updating your conda but it did not like the new syntax :-/

Reply:
yeah, something fishy is going on. I am sure it's something I have done incorrectly, but I haven't been able to track it down.

on a positive note, with my own machine the `conda activate` syntax works perfectly, so 👍 to you for showing me that.

Reply:
@jGaboardi as @Weathergod just reminded me you are creating and activating and env before you are installing things, so that won't activate the env vars! You can "re-activate" but in a CI you can just flip that and create the env with everything first and then activate it.

Reply:
@ocefpaf Do you mean something like moving the last two lines up two lines (see the bottom four lines below) should work with the new `conda activate` syntax?:

```
before_install:
  - wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
  - chmod +x miniconda.sh
  - ./miniconda.sh -b -p ./miniconda
  - export PATH=`pwd`/miniconda/bin:$PATH
  - conda create -y -q -n test-env python=$TRAVIS_PYTHON_VERSION
  - conda activate test-env
  - conda update --yes conda
  - conda config --add channels conda-forge
```

Reply:
As a side note, I see that you are fetching Miniconda-latest-Linux-x86_64.sh, unless something changed in the last few months, that still points to an ancient version of Miniconda as I discovered a little while back. They changed the name of Miniconda-latest-Linux-x86_64.sh to Miniconda2-latest-Linux-x86_64.sh (note the "2"), but only did so for newer versions.

Reply:
but, in relation to your issue, the problem is that any packages that gets installed won't get activated until you activate the environment again. So, activating the environment at this point before installation of packages won't necessarally work.

Reply:
@WeatherGod Thanks for the tips!

------------------------------------------
* miniconda issue
    * since this submodule of `pysal` is now only supporting Python 3.5+ I will want to be grabbing `Miniconda3-latest-Linux-x86_64.sh`, correct?

------------------------------------------

* travis.yml issue
    * I think I am a bit confused, but are you saying to move the `conda activate env` step from the `before_install` block down to the `install` block in the `yml`?

Reply:
* miniconda issue - actually, you can use either Miniconda3 or Miniconda2, all it means is that a different python version is *default*. You can still install py3k environments in Miniconda2 and py27 environments in Miniconda3.

* whatever it takes to activate the environment *after* install of packages. Just remember that if you are installing packages *before* activating the environment, but *after* creating it, then you need to explicitly state which environment you are installing packages into because the environment won't be active yet (done via the `-n ENVIRONMENT` argument)